_includes/icml/2020/sponsor.icml.es.html

```diff
@@ -0,0 +1,4 @@
+<object width="100%" 
```

Comment:
Until the new sponsor deck is available, and this information updates, this page should not be public.